deployment/terraform/iam.tf

```diff
@@ -192,3 +192,101 @@ resource "aws_iam_instance_profile" "batch_container_instance" {
   name = "${aws_iam_role.batch_container_instance_ec2.name}"
   role = "${aws_iam_role.batch_container_instance_ec2.name}"
 }
+
+#
+# Tilegarden executor
+#
+data "aws_iam_policy_document" "lambda_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+data "aws_iam_policy_document" "lambda_invoke" {
+  statement {
+    effect = "Allow"
+    resources = ["*"]
```

Comment:
Just want to confirm that this policy was originally scoped to the specific Lambda function in question, but that we're going to open it up to all resources because the Lambda function is defined in Claudia. I think this is fine given the deployment constraints we're under, but I also want to call out that I don't understand the security implications of it.

Reply:
Yeah, good point.  This would mean that rather than only being able to invoke itself, the Tilegarden Lambda would be able to invoke any other Lambda function.  Locking it down more would be better, but I think would require defining a new Terraform variable (so that the permission could say e.g. `resources = ["arn:aws:lambda:::function:${var.tilegarden_project_name}"]`) and instructing future deployers of new stacks to make sure the `PROJECT_NAME` in `.env` matches the Terraform variable.

Not a huge deal, but since I don't think there's any real risk from leaving the permission less restricted, I don't think it's worth the added complexity.

Reply:
Agreed 👍